lib/nimbusRecipe.ts

```diff
@@ -41,12 +42,40 @@ export class NimbusRecipe implements NimbusRecipeType {
     const featureValue = branch.features[0].value
 
     // XXX in this case we're really passing a feature value. Hmm....
-    const template = getTemplateFromMessage(featureValue)
-    branch.template = template
-    branchInfo.template = template
-    branchInfo.surface = getDisplayNameForTemplate(template)
+    // XXX about:welcome is special, so we have to account for it
+    let template;
+    (branch.features[0].featureId === "aboutwelcome" && branch.slug != 'control')
+      ? (template = "aboutwelcome")
+      : (template = getTemplateFromMessage(featureValue));
```

Comment:
If we can do this as `if/then/else` blocks, it would improve readbility.

Reply:
Also, why is `control` special cased here?  If it really should stay that way, it would be good to have a comment explaining the reasoning.

Reply:
Since we're just directly assigning the template instead of doing `getTemplateFromMessage`, the control branch ends up with a template instead of "none" in the table. (We can't use the function for about:welcome because it doesn't actually use the `template` property on its recipes. Go figure)